Source/WebKit/UIProcess/API/ios/WKWebViewIOS.mm

```diff
@@ -1323,6 +1323,18 @@ - (void)_updateOverlayRegions:(const WebKit::LayerPropertiesMap&)changedLayerPro
 
     scrollingCoordinatorProxy->updateOverlayRegionLayerIDs(overlayRegionsIDs);
 }
+
+- (void)_updateOverlayRegionsForCustomContentView
+{
+    if (![self _scrollViewCanHaveOverlayRegions:_scrollView.get()]) {
+        [_scrollView _updateOverlayRegionsBehavior:NO];
+        return;
+    }
+
+    [_scrollView _updateOverlayRegionsBehavior:YES];
+    HashSet<WebCore::IntRect> empty;
+    [_scrollView _updateOverlayRegionRects:empty];
```

Comment:
Random remark out of the scope of this PR, but tangentially related; could we change `-[WKBaseScrollView _updateOverlayRegionRects:]` to accept a const ref instead? Then you could pass in a temporary and make the call a little more succinct?

```cpp
@interface WKBaseScrollView : WKBEScrollView
- (void)_updateOverlayRegionRects:(const HashSet<WebCore::IntRect>&)overlayRegions;
@end

// ...
[_scrollView _updateOverlayRegionRects:{ }];
```

Reply:
that's a nice clean-up! updating the patch